### DISCUSSION TITLE
Got fatal ingestion error while trying to add new text into project

### DISCUSSION THREAD

**Discussion by lrmodesgh** [ManifoldScholar/manifold#3224] [Q&A]
Error Report from the Log displayed during ingestion of text :

```
Fatal Ingestion Error

The client application is unable to connect to the server's websocket. Please ensure that Manifold's "cable" service is available at the following location:

ws://desktop/cable

After the problem has been corrected, this ingestion can be resumed at the current URL.

DISMISS
```
i tried : 

`sudo manifold-ctl status`

and it shows : 

`run: cable: (pid 355012) 1953s; run: log: (pid 769) 41035s
run: client: (pid 355002) 1954s; run: log: (pid 759) 41035s
run: clockwork: (pid 355003) 1954s; run: log: (pid 773) 41035s
run: elasticsearch: (pid 355024) 1953s; run: log: (pid 761) 41035s
run: logrotate: (pid 354985) 1954s; run: log: (pid 762) 41035s
run: nginx: (pid 775) 41035s; run: log: (pid 765) 41035s
run: postgresql: (pid 763) 41035s; run: log: (pid 758) 41035s
run: puma: (pid 355011) 1953s; run: log: (pid 774) 41035s
run: redis: (pid 764) 41035s; run: log: (pid 760) 41035s
run: sidekiq: (pid 355004) 1954s; run: log: (pid 776) 41035s`

please help me resolving the above issue. I have installed manifold as explained in the documentation. (used the package, not the docker image)

**Comment by zdavis** [comment]
It looks like Manifold isn't running on a FQDN or IP, and the client application isn't able to connect to the web socket service. You might take a look at your browser console and see if there's an error message that explains why it can't talk to the websocket service. Or, you might try visiting https://desktop/cable — if that "desktop" domain is resolving correctly and the cable service is responding, you'll see a white screen and "page not found," indicating that the cable service is listening at the path as it should be.

**Comment by zdavis** [comment]
Try changing the external_url value in /etc/config/manifold.rb to "http://127.0.0.1", restarting all the services, and then accessing at that URL in the browser.

**Reply by lrmodesgh** [reply]
Hey @zdavis , thanks for getting back. as you said, after changing the external_url to "localhost" worked out fine. the loop back address didnt worked out.

Thanks a lot for replying back. I am trying to use manifold at French Institute Pondicherry, for a historical project publications. This would help me prove the reliability of FOSS ecosystem, collaborative practice to humanities scholars here.
